resources/dicts/patrols/plains/border/greenleaf.json

```diff
@@ -31,13 +36,4753 @@
+                "exp": 0,
+                "weight": 10,
+                "art": "gen_coyote_danger2_w1",
+                "injury": [
+                    {
+                        "cats": ["r_c"],
+                        "injuries": ["big_bite_injury"],
+                        "scars": ["BRIGHTHEART"]
+                    }
+                ],
+                "history_text": {
+                    "scar": "This cat is scarred from bravely defending the Clan against a coyote.",
+                    "reg_death": "This cat died to keep the Clan safe from a large coyote.",
+                    "lead_death": "died to keep c_n safe from a large coyote"
+                },
+                "relationships": [
+                    {
+                        "cats_to": ["r_c"],
+                        "cats_from": ["clan"],
+                        "mutual": false,
+                        "values": ["respect", "trust"],
+                        "amount": 5
+                    }
+                ]
+            }
+        ]
+    },
+    {
+        "patrol_id": "pln_bord_greenleaf_warriortunnel1",
+        "biome": ["plains"],
+        "season": ["greenleaf"],
+        "types": ["border"],
+        "tags": [],
+        "patrol_art": "pln_bord_tunnel",
+        "min_cats": 3,
+        "max_cats": 6,
+        "min_max_status": {
+            "apprentice": [-1, -1],
+            "normal adult": [1, 6]
+        },
+        "weight": 20,
+        "intro_text": "Patrolling the border and updating the markings, r_c calls out to the rest of the patrol, making p_l stand up on {PRONOUN/p_l/poss} hind legs to peek over the grass. r_c has found an opening to what seems to be a new set of tunnels.",
+        "decline_text": "The patrol decides not to check the tunnels. After all, who knows what might be hiding inside?",
+        "chance_of_success": 40,
+        "success_outcomes": [
+            {
+                "text": "The cicadas are so dense and noisy here it's almost hard to talk. This new (or newly found) tunnel might be useful, but it'll need proper investigation, and if c_n wants it, it's on the wrong side of the border. Worth reporting though!",
+                "exp": 20,
+                "weight": 10,
+                "art": "gen_train_talkingWW",
+                "relationships": [
+                    {
+                        "cats_to": ["patrol"],
+                        "cats_from": ["patrol"],
+                        "mutual": false,
+                        "values": ["comfort", "trust"],
+                        "amount": 5
+                    }
+                ]
+            },
+            {
+                "text": "It's always valuable to have safe place to retreat to near c_n's borders, s_c points out. {PRONOUN/s_c/subject/CAP} {VERB/s_c/check/checks} over the tunnel - the scents inside are long stale, but the construction is still good. The patrol carefully marks the burrow. One day, in danger, it might save a Clanmate's life.",
```

Comment:
```suggestion
                "text": "It's always valuable to have a safe place to retreat to near c_n's borders, s_c points out. {PRONOUN/s_c/subject/CAP} {VERB/s_c/check/checks} over the tunnel - the scents inside are long stale, but the construction is still good. The patrol carefully marks the burrow. One day, in danger, it might save a Clanmate's life.",
```